src/styles/blocks/page.scss

```diff
@@ -0,0 +1,5 @@
+body {
+  display: flex;
+  margin: 0;
+  height: 100vh;
```

Comment:
To center the stopwatch, you need to add 'justify-content: center;' and 'align-items: center;' to the body styles.